### ISSUE TITLE
How long did it take to show up your results on KITTI leaderboard

### ISSUE THREAD

**Issue by MichaelWangGo** [David-Zhao-1997/High-frequency-Stereo-Matching-Network#15]
Hi authors, 

Thank you for your great work, I am a beginner in this field, and also want to submit my results to KITTI and Middlebury evaluation server. But several questions confused me a lot. Can you tell me how to deal with the different sizes of KITTI images (some are 1242x375 and some are 1241x376) and Middlebury images, do I need to crop the predicted disparity results into the same size or resize them into the original sizes? And how long did it take to show up your ranking on both leaderboards.

**Comment by David-Zhao-1997** [comment]
1. Generally speaking, we use padding technique to solve the image size problem, before entering the neural network, we padding the network to the size that the network can handle, then we perform inference or training, after inference is done, then we perform unpad operation to remove these parts. You can find this code directly in core/utils/utils.py in InputPadder.

        image1 = image1[None].cuda()
        image2 = image2[None].cuda()

        padder = InputPadder(image1.shape, divis_by=32)
        image1, image2 = padder.pad(image1, image2)

        with autocast(enabled=mixed_prec):
            _, flow_pr = model(image1, image2, iters=iters, test_mode=True)
        flow_pr = padder.unpad(flow_pr.float()).cpu().squeeze(0)

2. 
- Evaluation on the KITTI leaderboard is very fast, usually it takes a few seconds to get the results, and you can choose to make your results public or not.
- The images in the Middlebury dataset are of very high resolution, and it usually takes about 20 minutes to transfer and evaluate, and you'll get the results of the training set after the transfer and evaluation is complete. If you want to know the test set results (you have one and only one chance per method), you can choose to do a test set evaluation, which consumes roughly another 20 minutes and is manually verified after the test is completed; you do not see the results of your method's evaluation. In about 1 day or so, the results will be posted on the leaderboard, and you may receive an email at the same time.

**If you have any other questions, feel free to contact me.**

**Comment by MichaelWangGo** [comment]
Thank you very much@David-Zhao-1997
